java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -176,7 +219,35 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_fromScalar(JNIEnv *env,
     cudf::mask_state mask_state =
         scalar_val->is_valid() ? cudf::mask_state::UNALLOCATED : cudf::mask_state::ALL_NULL;
     std::unique_ptr<cudf::column> col;
-    if (row_count == 0) {
+    if (dtype.id() == cudf::type_id::LIST) {
+      // Neither 'cudf::make_empty_column' nor 'cudf::make_column_from_scalar' supports
+      // LIST type for now (04/27/2021), so the list precedes the others and takes care
```

Comment:
Rather than listing a date, I'd much rather see a link to the corresponding cudf issue requesting the feature.

Reply:
Good suggestion, updated.